src/models/Event/Event.test.ts

```diff
@@ -0,0 +1,12 @@
+import {getEventList} from "./Event";
+
+describe("Log Event", () => {
+    test("if Event list properly adds/removes elements", () => {
```

Comment:
Perhaps simplify to `describe("getEventList", () => { test("properly adds/removes elements", () => {...`

Reply:
done all comments